packages/client/hmi-client/src/components/workflow/tera-calibration-node-ciemss.vue

```diff
@@ -237,20 +239,49 @@ const runCalibrate = async () => {
 	};
 	const response = await makeCalibrateJobCiemss(calibrationRequest);
 
+	// Run MQ Test
+	console.log(`Checking MQ for job Id ${response.simulationId}`);
+	const auth = useAuthStore();
+	const MQPoller = new EventSourcePolyfill(
+		`/api/simulations/${response.simulationId}/partial-result`,
```

Comment:
Maybe rename `MQPoller` to something else to avoid confusion. We are not performing a polling-action or a request per se, it is sitting there waiting for messages to arrive from the sever-side.